client/src/css/app.sass

```diff
@@ -1,105 +1,179 @@
 // app global css in Sass form
 @font-face
-  font-family: "Atkinson"
-  src: url(./fonts/Atkinson_Hyperlegible/AtkinsonHyperlegible-Regular.ttf)
+    font-family: "Atkinson"
+    src: url(./fonts/Atkinson_Hyperlegible/AtkinsonHyperlegible-Regular.ttf)
 
 html
     font-family: "Atkinson"
 
-.text--grey
-    color: $dark-grey
-
-.accent
-    background: $accent
-
-.active
-    background: $active
-
-.alert
-    background-color: #eee
-    border-radius: 4px
-    margin: 16px 0
-    padding: 16px 16px 16px 16px
-    font-size: 1em
-    border-width: 0 5px
-    border-style: solid
-    border-color: $dark-grey
-    letter-spacing: 0.5px
-    .alert-title
-        margin: -16px -16px 16px -16px
-        padding: 4px 8px
-        font-weight: bold
-        border-bottom: 1px solid
-
-.alert-tip
-    background-color: $active
-    border-color: $active
-
-    .alert-title
-        font-weight: bold
-        border-color: $secondary
-
-.alert-success
-    background-color: #c1f4c5
-    border-color: $positive
-
-    .alert-title
-        font-weight: bold
+.body--light
+    .text--grey
+        color: $dark-grey
+
+    .accent
+        background: $accent
+
+    .active
+        background: $active
+
+    .alert
+        background-color: #eee
+        border-radius: 4px
+        margin: 16px 0
+        padding: 16px 16px 16px 16px
```

Comment:
For styles that are only meant to handle light mode and dark mode, I would avoid including any that involve changes to dimensions to elements, like padding, margin, etc. This practice has resulted in introducing different layouts and magnifying the alignment problem on the login page for dark mode. 

![login-form](https://user-images.githubusercontent.com/2668987/213011401-953017a2-dd11-483f-81ba-cb536de5b3ea.png)